app/lib/features/chat_ng/widgets/chat_bubble.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:acter/common/themes/acter_theme.dart';
+import 'package:flutter/material.dart';
+import 'package:acter/common/extensions/options.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+
+// Chat Bubble UI
+class ChatBubble extends StatelessWidget {
+  final int? messageWidth;
+  // inner content
+  final Widget child;
+  final bool isUser;
+  final bool showAvatar;
+  final bool wasEdited;
+
+  const ChatBubble({
+    super.key,
+    required this.child,
+    this.messageWidth,
+    this.isUser = false,
+    this.showAvatar = true,
```

Comment:
Form the code `showAvatar` seems to only decide whether what the bottom corner looks like. But that isn't really what `showAvatar` implies as semantic-information. For e.g. in a 1-on-1 DM there is a good reason to not show either avatar (so `showAvatar = false`), while we still want to show the bottom corner differently some times ... we should rename that field to more appropriately describe what this is about.